config/src/main/resources/classic.json

```diff
@@ -13,10 +13,12 @@
     "thanosBlock": 11700000,
     "magnetoBlock": 13189133,
     "mystiqueBlock": 14525000,
+    "spiralBlock": 19250000,
     "ethash": {
 
     },
     "discovery" : {
+      "dns": "enrtree://AJE62Q4DUX4QMMXEHCSSCSC65TDHZYSMONSD64P3WULVLSF6MRQ3K@all.classic.blockd.info",
```

Comment:
Do you think this deserve an entry in the changelog too?

Reply:
I think we're ok without it. Hopefully there should be no difference for the users, I'm not against adding it though